### DISCUSSION TITLE
menubar icon looks blur on macOS monterey

### DISCUSSION THREAD

**Discussion by leewi9** [pilotmoon/Scroll-Reverser#148] [General]
menubar icon looks blur on macOS monterey

**Comment by AaronBuxbaum** [comment]
+1 -- this sounds like a small concern, but it really does stick out

**Comment by pilotmoon** [comment]
@leewi9 , @AaronBuxbaum Is this blurry icon thing still an issue on latest OS? It looks OK on my monitor. 
If it's still bad for you, you post a screenshot to show what you mean? And confirm the Scroll Reverser version and OS version you are using.

**Reply by AaronBuxbaum** [reply]
it looks good to me now!
